mindjunkies/lecture/tests/test_views.py

```
@@ -133,3 +243,270 @@ def test_mark_lecture_complete(client, user, lecture):
+
+    with pytest.raises(ValueError):
+        view.get_form_class()
+
+
+@pytest.mark.django_db
+def test_form_valid_saves_content_correctly(factory, teacher_user, course, lecture):
+    request = factory.get('/')
+    request = add_middleware_to_request(request)
+    request.user = teacher_user
+
+    form = LecturePDFForm(data={'pdf_title': 'Test PDF'}, files={})
```

Ruff (0.8.2)

Local variable `form` is assigned to but never used

Remove assignment to unused variable `form`

(F841)